x/blobstream/types/genesis.go

```diff
@@ -9,12 +9,12 @@ import (
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
-// DefaultParamspace defines the default qgb module parameter subspace
+// DefaultParamspace defines the default blobstream module parameter subspace
```

Comment:
```suggestion
// DefaultParamspace defines the default Blobstream module parameter subspace
```

I realize that some of these comments are for "blobstream module", and understand these suggestions may need to be ignored if it should be lowercase

Reply:
true :+1: